food-truck-frontend/src/components/dashboards/user/react/UserDashboardComponent.tsx

```diff
@@ -101,6 +101,17 @@ class UserDashboardComponent extends Component<
               </Card>
             ) : null}
 
+            {/* Go to owner dashboard */}
```

Comment:
Was that comment supposed to come with the rest?

Reply:
No.